src/scss/_buy.scss

```diff
@@ -6,15 +6,18 @@
   @media (min-width: $bp-f) {
     padding: 12em 0;
   }
-  
+
   &__type {
     @extend .type__p--reverse;
     text-align: center;
     margin: 1em auto 2em;
     width: 90%;
 
     @media (min-width: $bp-b) {
+        margin: 1.5em auto 3em;
```

Comment:
This should only be two spaces, it looks like. I'm a stickler on inconsistent spacing. 😉